src/main/java/tc/oc/pgm/commands/MapCommands.java

```diff
@@ -27,43 +26,53 @@
 import tc.oc.pgm.AllTranslations;
 import tc.oc.pgm.api.Permissions;
 import tc.oc.pgm.api.chat.Audience;
-import tc.oc.pgm.api.match.MatchManager;
+import tc.oc.pgm.api.map.Contributor;
+import tc.oc.pgm.api.map.MapInfo;
+import tc.oc.pgm.api.map.MapLibrary;
+import tc.oc.pgm.api.map.MapTag;
 import tc.oc.pgm.commands.annotations.Text;
-import tc.oc.pgm.map.Contributor;
-import tc.oc.pgm.map.MapInfo;
-import tc.oc.pgm.map.MapLibrary;
-import tc.oc.pgm.map.MapPersistentContext;
-import tc.oc.pgm.map.PGMMap;
-import tc.oc.pgm.maptag.MapTag;
-import tc.oc.pgm.maptag.MapTagsCondition;
+import tc.oc.pgm.rotation.MapOrder;
 import tc.oc.pgm.util.PrettyPaginatedResult;
 import tc.oc.util.components.ComponentUtils;
 import tc.oc.util.components.Components;
 
 public class MapCommands {
 
+  @Command(
+      aliases = {"loadnewmaps"},
+      desc = "Loads new maps and outputs any errors")
+  public static void loadNewMaps(MapLibrary library, @Switch('f') boolean force) {
+    library.loadNewMaps(force); // MapLibrary will handle sending output asynchronously
+  }
+
   @Command(
       aliases = {"maplist", "maps", "ml"},
       desc = "Shows the maps that are currently loaded",
-      usage = "[-a <author>] [-p <page>] [[!]#<maptag>...]",
+      usage = "[-a <author>] [-p <page>] [-# <tag1>,<tag2>]",
       help =
           "Shows all the maps that are currently loaded including ones that are not in the rotation.")
   public static void maplist(
       Audience audience,
       CommandSender sender,
       MapLibrary library,
-      MapTagsCondition mapTags,
+      @Fallback(Type.NULL) @Switch('t') String tags,
       @Fallback(Type.NULL) @Switch('a') String author,
       @Fallback(Type.NULL) @Switch('p') Integer page)
       throws CommandException {
     if (page == null) page = 1;
 
-    Stream<PGMMap> search = library.getMaps().stream().filter(mapTags);
+    Stream<MapInfo> search = Sets.newHashSet(library.getMaps()).stream();
+    if (tags != null) {
+      final Set<String> tagSet =
+          Stream.of(tags.split(",")).map(String::trim).collect(Collectors.toSet());
+      search = search.filter(map -> matchesTags(map, tagSet));
```

Comment:
People can no longer negate maptags as @Pablete1234 suggested me to do. Let's if people would want to search for all DTM maps except the ones mixed with cores (SSB) - `/maps #monument !#core`